catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -145,19 +151,36 @@ public UpdateResponse version(UpdateResponse updateResponse)
         }
         setSkipFlag(updateResponse);
 
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("Versioning updated metacards: " + updateResponse.getUpdatedMetacards());
```

Comment:
I'm wondering if it is meaningful to print out `getUpdatedMetacards`.  Is it meaningful enough to just have the number of metacards that are being versioned?  We can probably remove `isTraceEnabled` if that is the case,